### DISCUSSION TITLE
Why only boolean assert have userMessage parameter?

### DISCUSSION THREAD

**Discussion by Aedjis** [xunit/xunit#2408] [Ideas]
Hello there.

I'm wandering why only the true and false assert have the possibility to avec a userMessage parameter. 
Since sometime we will assert something for a reason but the reason behind it was not obvious, a userMessage will be nice.
Indid we can juste use the boolean assert but generaly it's the worst for have as many as possible information.

thanks for your reading and your answord.

**Comment by bradwilson** [comment]
https://github.com/xunit/xunit/issues/350#issuecomment-78211019
